cli/install/install.go

```diff
@@ -492,30 +552,33 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 	} else {
 		log.Infof("Downloading tt...")
-		err = downloadRepo(search.GitRepoTT, ttVersion, path, logFile, installCtx.verbose)
+		if versionFound {
+			err = downloadRepo(search.GitRepoTT, ttVersion, path, logFile, installCtx.verbose)
+		} else {
+			err = downloadRepo(search.GitRepoTT, "master", path, logFile, installCtx.verbose)
+			util.ExecuteCommand("git", installCtx.verbose, logFile, path, "checkout", ttVersion)
+		}
 	}
 
 	if err != nil {
 		printLog(logFile.Name())
 		return err
 	}
+
 	// Build tt.
 	log.Infof("Building tt...")
-	err = util.ExecuteCommand("mage", installCtx.verbose, logFile, path,
-		"build")
+	err = util.ExecuteCommand("mage", installCtx.verbose, logFile, path, "build")
 	if err != nil {
 		printLog(logFile.Name())
 		return err
 	}
-
 	// Copy binary.
 	log.Infof("Copying executable...")
 	err = copyBuildedTT(binDir, path, versionStr, installCtx, logFile)
 	if err != nil {
 		printLog(logFile.Name())
 		return err
 	}
-
```

Comment:
This change is not related to the feature implementation.

Reply:
fixed

Reply:
Looks like, not fixed yet.